docs/examples/prerequisites.mdx

```diff
@@ -30,7 +30,7 @@ Current stable release: `v1.1`
 <tr>
 <td>
 
-Linux users should generally install `GCC` and `Clang`, according to their distribution’s documentation.
+Linux users should generally install `GCC` and `Clang`, according to their distribution`s documentation.
```

Comment:
The back tick is in markdown syntax for `inline code` blocks. I'd left the previous one.